app/manager-frontend/src/pages/UserHome.jsx

```diff
@@ -4,19 +4,25 @@ import { ActivitySourcesList } from "./ActivitySource/list";
 import { useParams, useNavigate } from "react-router-dom";
 import { useEffect } from "react";
 import { AgentsList } from "./Agent/list";
+import { UsersList } from "./User/list";
 
 export const UserHome = () => {
   const homepageActions = [
     {
       title: "Activity Source",
```

Comment:
Could this be "Activity Sources" instead? since we have "Agents" and "Users" as titles

Reply:
Yep. I will update it now as suggested.